components/global/cliTab.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<div>
+		<div class="flex flex-row m-auto w-full mt-32px mb-24px border-b border-b-grey-10">
+			<li v-for="tab of tabs" :key="tab.id" class="mr-24px !list-none last-of-type:mr-0">
+				<button class="pb-10px flex items-center has-icon" :class="activeTab === tab.id ? 'active' : ''" @click="switchTabs(tab.id)">
+					<!-- <img :src="require(`~/assets/images/${tab.id}.svg`)" :alt="tab.label" class="h-16px w-16px mr-16px !my-0" /> -->
+					<span class="text-14 text-left text-black tracking-[0.02em] mobile:min-w-[80px]">{{ tab.label }}</span>
+				</button>
+			</li>
+		</div>
+		<div>
+			<nuxt-content :document="pageData"></nuxt-content>
+		</div>
+	</div>
+</template>
+<script>
+export default {
+	layout: 'docs',
+	data() {
+		return {
+			pageData: '',
+			tabs: [
+				{ label: 'Mac', id: 'mac' },
+				{ label: 'Linux', id: 'linux' },
+				{ label: 'Windows', id: 'windows' },
+                { label: 'Source', id: 'source'}
+			],
+			activeTab: 'mac'
+		};
+	},
+	mounted() {
+		this.fetchPageData('cli-mac');
```

Comment:
@Youngestdev this should be `this.switchTabs('mac')` as well.

Reply:
ok